test/pagination/offset-paging.js

```diff
@@ -777,3 +777,54 @@ test('should handle an interface type', async t => {
   }
   t.deepEqual(expect, data.user.writtenMaterial)
 })
+
+test('should not allow pagination with greater than pageSizeLimit at the root', async t => {
+  const query = makeUsersQuery({ first: 1000 })
+  const { data, errors } = await run(query)
+
+  t.deepEqual(data.users, null)
+  t.deepEqual(errors[0].message, 'Maximum page size of User type, is 100')
+})
+
+test('should not allow nestedpagination with greater than pageSizeLimit', async t => {
+  const query = `{
+    users(first: 2) {
+      edges {
+        node {
+          fullName,
+          comments(first: 1000) {
+            total
+            edges {
+              node { body }
+            }
+          }
+        }
+      }
+    }
+  }`
+  const { data, errors } = await run(query)
+  t.deepEqual(data.users, null)
+  t.deepEqual(errors[0].message, 'Maximum page size of Comment type, is 100')
+})
+
+test('should use default page limit if set', async t => {
+  const query = `{
+    users {
+      edges{
+        node{
+          id
+          comments {
+            edges{
+              node{
+                id
+              }
+            }
+          }
+        }
+      }
+    }
+  }`
+  const { data, errors } = await run(query)
+  errCheck(t, errors)
+  t.deepEqual(data.users.edges[0].node.comments.edges.length, 3)
```

Comment:
Should this be 2? `sqlDefaultPageSize: 2,`

Reply:
thanks, was a bug in the page implementation. fixed now.